0.4/schemas/image.schema

```diff
@@ -0,0 +1,258 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://ngff.openmicroscopy.org/0.3/schemas/image.schema",
+  "title": "NGFF Image",
+  "description": "JSON from OME-NGFF .zattrs",
+  "type": "object",
+  "properties": {
+    "multiscales": {
+      "description": "The multiscale datasets for this image",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "datasets": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "type": "object",
+              "properties": {
+                "path": {
+                  "type": "string"
+                },
+                "coordinateTransformations": {
```

Comment:
I think that coordinate transformations should be moved to a separate schema.

Reply:
Agreed, separating `axes` and `coordinateTransformations`  matches the way the specification is written and should increase the readability and the management of these schemas.

A technical side, as these schemas are not yet published (and we need to have some deeper discussions around URL as well as packaging - see https://github.com/ome/ngff/pull/77), I suspect handling separate JSON schema files will require to make some decisions as relative references are not resolvable. Since these concepts are only used within the context of `multiscales` in v0.4, my inclination would be to start by defining internal sub-schemas using `#defs` - see https://github.com/ome/ngff/pull/87#pullrequestreview-869054310. As the intent is to use these concepts in other portions in the specification, I propose we review this assumption and define `axes.schema` and `coordinateTransformations.schema` when this becomes a requirement.

Reply:
I agree, for now using them to `#defs` in the same schema is probably the best solution.